homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -296,25 +307,36 @@
     ATTR_MODE: "mode",
     ATTR_BUZZER: "buzzer",
     ATTR_CHILD_LOCK: "child_lock",
-    ATTR_TARGET_HUMIDITY: "target_humidity",
     ATTR_LED_BRIGHTNESS: "led_brightness",
-    ATTR_USE_TIME: "use_time",
-    ATTR_HARDWARE_VERSION: "hardware_version",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_TARGET_HUMIDITY: "target_humidity",
+    ATTR_HARDWARE_VERSION: "hardware_version",
+    ATTR_USE_TIME: "use_time",
     ATTR_TRANS_LEVEL: "trans_level",
     ATTR_BUTTON_PRESSED: "button_pressed",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_CA_AND_CB = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_TARGET_HUMIDITY: "target_humidity",
+    ATTR_HARDWARE_VERSION: "hardware_version",
+    ATTR_USE_TIME: "use_time",
     ATTR_MOTOR_SPEED: "motor_speed",
     ATTR_DEPTH: "depth",
     ATTR_DRY: "dry",
 }
 
+AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_JSQ = {
+    **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_LED: "led",
+    ATTR_NO_WATER: "no_water",
+    ATTR_LID_OPENED: "lid_opened",
```

Comment:
`lid_opened` seems like it should be a binary sensor. We want to minimize state attributes and instead break out to as many individual sensors as possible.

Reply:
Yes, it could be a binary sensor. Could you guide me with some example of how I could replace with sate with a binary sensor?